src/App.js

```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
+import React from 'react';
+import {BrowserRouter as Router, Routes, Route, Link} from 'react-router-dom';
 import './App.css';
+import HomePage from './Pages/HomePage';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
```

Comment:
I would prefer always adding className before anything else and keeping that consistent throughout the app;
EX: <img className=“...” src=“...” alt=“...” />

Reply:
Unable to see image example, image won't load. The app file currently does start with classname

Reply:
Sorry, the example is : <img  className="App-logo"src={logo} alt="logo" />
It’s best to always add the className or Id in the beginning of each tag, as seen above. This should be consistent across any tag that has a className or Id.